test/integration/targets/include_vars/tasks/main.yml

```diff
@@ -65,12 +65,12 @@
       - "testing == 456"
       - "base_dir == 'services'"
       - "webapp_containers == 10"
-      - "{{ include_every_dir.ansible_included_var_files | length }} == 7"
+      - "{{ include_every_dir.ansible_included_var_files | length }} == 9"
```

Comment:
why do existing tests need to change? 

shouldn't this be transparent when not setting the value to 'true'?

Reply:
That's because there is a task above that loads all the YAML files in `vars` directory and the added files (`vars/hashes/*.yml`) are loaded at that point as well.

https://github.com/ansible/ansible/blob/0bd39916c537600b861967158c4e9d4d9a42483b/test/integration/targets/include_vars/tasks/main.yml#L56-L60

It does not affect the correctness of the tests, so I just kept the new files in `vars` directory and updated the existing assertions. But if you don't like that we have to update them, we can move the files to a different directory, such as `vars2/hashes/*.yml`. What do you think?

Reply:
@bcoca Any thoughts? I can move the new fixture files to a different directory and revert the existing assertions if that's what you prefer it.

Reply:
Reverted the existing test cases.